scanpipe/models.py

```diff
@@ -1948,7 +1948,7 @@ def has_directory_content_fingerprint(self):
 
     def elfs(self):
         """
-        Resources that are ``files`` and their filetype starts with "elf" and
+        Resources that are ``files`` and their filetype starts with "ELF" and
```

Comment:
This change is not correct, the string is still "elf"

Reply:
Done!